styles.css

```diff
@@ -27,14 +27,26 @@ nav {
 
 .navbar-menu {
   display: none;
+  z-index: 10;
+  background-color: #3c3a39;
+  position: absolute;
+  width: 100%;
+  left: 0;
+  font-size: 32px;
+  line-height: 44px;
+  top: 72px;
+  height: 755px;
 }
 
 .navbar-link {
   list-style: none;
-  line-height: 20px;
-  font-size: 15px;
-  font-weight: 600;
-  margin: 0 12px;
+  line-height: 44px;
+  font-size: 32px;
+  font-weight: 700;
+  color: #fff5e1;
```

Comment:
- [ ] I can see you did set the correct text colour here. However, it doesn't take effect in your design. I want to suggest setting it to the `a` tag.

See example:
```css
.navbar-link > a {
    color: #fff5e1;
}
```

Please take a look at these screenshots to understand the point am making here.

Your Design | Figma Design |
:---------------:|:-------------------:|
![a1](https://user-images.githubusercontent.com/39523312/161654026-c366784c-50ec-442f-a681-12585e8c49f6.png)|![a2](https://user-images.githubusercontent.com/39523312/161654029-23daf4ee-90d9-4f89-9ac0-f9ad221c8149.png)